README.md

```diff
@@ -7,6 +7,7 @@
 # Sentry for PHP
 
 [![Build Status](https://secure.travis-ci.org/getsentry/sentry-php.png?branch=master)](http://travis-ci.org/getsentry/sentry-php)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/getsentry/sentry-php)](https://ci.appveyor.com/project/sentry/sentry-php)
```

Comment:
This one seems to match the other ones.

```suggestion
[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/getsentry/sentry-php?branch=master&svg=true)](https://ci.appveyor.com/project/sentry/sentry-php)
```

It's weird however it reports as failed...